controller/src/main/java/org/jboss/as/controller/operations/validation/OperationValidator.java

```diff
@@ -130,6 +130,7 @@ public void validateOperation(final ModelNode operation) {
         OperationEntry entry = root.getOperationEntry(address, name);
         if (entry == null) {
             throwOrWarnAboutDescriptorProblem(ControllerLogger.ROOT_LOGGER.noOperationEntry(name, address));
+            return; // in case of warning no further processing possible (entry causes NullPointer)
```

Comment:
@kabir Could you have a look at this fix, please? Many other points in the code have the pattern
```
throwOrWarnAboutDescriptorProblem(ControllerLogger....);
return;
```
Here the `return;` is missing. Is it intended to break later with an NPE in case of logging a warning?

Reply:
@boris-unckel I think it looks fine. A lot of this is cruft from ancient times before we became a lot stricter about needing definitions for everything